packages/account/src/Components/personal-details/__tests__/personal-details.spec.js

```diff
@@ -751,6 +849,10 @@ describe('<PersonalDetails/>', () => {
             value: {
                 ...props.value,
                 tax_residence: 'France',
+                document_type: {
+                    value: 'national_id',
+                    text: 'National ID',
```

Comment:
I would agree, but here we just added small piece of code to the big component and added tests to this also big file.
we can consider it while refactoring this component to ts in upcoming sprints

Reply:
@maryia-deriv, I'm not sure that I got your point :)

Reply:
we cant do all possible refactoring in one card, first we should deliver the functionally described in the card. 
Otherwise we will have to much files changed, which will be difficult to review.
for refactoring purposes we have a separate cards, which can be planned separately in order to not delay feature releases

Reply:
const national_id_value = 'national_id';
const national_id_text = 'National ID';
I only suggest adding these 2 variables. You guys have already added a lot of changes to this file including typing these strings multiple times. :)